[2024-10-03 08:12:48.9695] [INFO] [D:\a\_work\1\s\src\modules\launcher\PowerLauncher\App.xaml.cs::55]
Starting PowerToys Run with PID=29492

[2024-10-03 08:12:48.9984] [INFO] [D:\a\_work\1\s\src\modules\launcher\PowerLauncher\App.xaml.cs::66]
Runner pid=24200

[2024-10-03 08:12:49.3146] [INFO] [D:\a\_work\1\s\src\modules\launcher\PowerLauncher\App.xaml.cs::108]
On Startup.

[2024-10-03 08:12:49.4299] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Stopwatch.cs::44]
ImageLoader.Initialize - Preload images cost <4ms>

[2024-10-03 08:12:49.4299] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Image\ImageLoader.cs::99]
Number of preload images is <109>, Images Number: 2, Unique Items 2

[2024-10-03 08:12:49.4864] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Stopwatch.cs::44]
|Wox.Infrastructure.Alphabet.Initialize|Preload pinyin cache <32ms>

[2024-10-03 08:12:49.4864] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Alphabet.cs::46]
Number of preload pinyin combination<0>

[2024-10-03 08:12:49.6903] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <Calculator> is <34ms>

[2024-10-03 08:12:49.7117] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <Folder> is <2ms>

[2024-10-03 08:12:49.7117] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <History> is <0ms>

[2024-10-03 08:12:49.7117] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <Windows Indexer> is <0ms>

[2024-10-03 08:12:49.7117] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <OneNote> is <0ms>

[2024-10-03 08:12:49.7203] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <PowerToys> is <1ms>

[2024-10-03 08:12:49.9404] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <Program> is <218ms>

[2024-10-03 08:12:49.9424] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <Registry> is <0ms>

[2024-10-03 08:12:49.9622] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <Service> is <18ms>

[2024-10-03 08:12:49.9685] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <Shell> is <5ms>

[2024-10-03 08:12:49.9685] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <System Commands> is <2ms>

[2024-10-03 08:12:49.9685] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <Time and date> is <0ms>

[2024-10-03 08:12:49.9685] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <Unit Converter> is <0ms>

[2024-10-03 08:12:49.9685] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <Windows Uri Handler> is <1ms>

[2024-10-03 08:12:49.9685] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <ValueGenerator> is <0ms>

[2024-10-03 08:12:50.0202] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <VS Code Workspaces> is <38ms>

[2024-10-03 08:12:50.0202] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <Web Search> is <0ms>

[2024-10-03 08:12:50.0202] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <Windows Settings> is <1ms>

[2024-10-03 08:12:50.0466] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <Windows Terminal> is <20ms>

[2024-10-03 08:12:50.0734] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <Window Walker> is <1ms>

[2024-10-03 08:12:50.6373] [INFO] [D:\a\_work\1\s\src\modules\launcher\PowerLauncher\SettingsReader.cs::87]
Successfully read new settings. retryCount=1

[2024-10-03 08:12:50.6373] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Storage\JsonStorage`1.cs::139]
Saving cached data at <C:\Users\LENOVO LEGION\AppData\Local\Microsoft\PowerToys\PowerToys Run\Settings\Plugins\Microsoft.Plugin.Shell\ShellPluginSettings.json>

[2024-10-03 08:12:50.6526] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <Registry> is <0ms>

[2024-10-03 08:12:50.6526] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <Shell> is <5ms>

[2024-10-03 08:12:50.6526] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <Time and date> is <0ms>

[2024-10-03 08:12:50.6526] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <History> is <0ms>

[2024-10-03 08:12:50.6526] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <ValueGenerator> is <0ms>

[2024-10-03 08:12:50.6526] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <Folder> is <4ms>

[2024-10-03 08:12:50.6526] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <Service> is <18ms>

[2024-10-03 08:12:50.6526] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <Unit Converter> is <0ms>

[2024-10-03 08:12:50.6526] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <Windows Uri Handler> is <1ms>

[2024-10-03 08:12:50.6526] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <System Commands> is <2ms>

[2024-10-03 08:12:50.6526] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\Common\DefaultBrowserInfo.cs::74]
I've tried updating the chosen Web Browser info at least once.

[2024-10-03 08:12:50.6526] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <Window Walker> is <1ms>

[2024-10-03 08:12:50.6526] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <Windows Indexer> is <2ms>

[2024-10-03 08:12:50.6526] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <Calculator> is <34ms>

[2024-10-03 08:12:50.6526] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <Web Search> is <5ms>

[2024-10-03 08:12:50.6526] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <PowerToys> is <13ms>

[2024-10-03 08:12:50.6526] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <Windows Settings> is <13ms>

[2024-10-03 08:12:51.8587] [INFO] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Storage\PackageRepository.cs::111]
Indexed 45 packaged applications

[2024-10-03 08:12:51.8587] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Stopwatch.cs::44]
Microsoft.Plugin.Program.Main - Package index cost <1120ms>

[2024-10-03 08:12:52.2482] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\ShellLinkHelper.cs::169]
-------------------------- Begin exception --------------------------
Message: Failed to fetch description for C:\Windows\syswow64\odbcad32.exe, Error HRESULT E_FAIL has been returned from a call to a COM component.

Exception full name  : System.Runtime.InteropServices.COMException
Exception message    : Error HRESULT E_FAIL has been returned from a call to a COM component.
Exception stack trace:
   at Wox.Infrastructure.ShellLinkHelper.IShellLinkW.GetDescription(StringBuilder pszName, Int32 cchMaxName)
   at Wox.Infrastructure.ShellLinkHelper.RetrieveTargetPath(String path)
Exception source     : Wox.Infrastructure
Exception target site: Void GetDescription(System.Text.StringBuilder, Int32)
Exception HResult    : -2147467259
-------------------------- End exception --------------------------


[2024-10-03 08:12:52.2714] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\ShellLinkHelper.cs::169]
-------------------------- Begin exception --------------------------
Message: Failed to fetch description for C:\PROGRA~2\WINDOW~4\wmplayer.exe, Error HRESULT E_FAIL has been returned from a call to a COM component.

Exception full name  : System.Runtime.InteropServices.COMException
Exception message    : Error HRESULT E_FAIL has been returned from a call to a COM component.
Exception stack trace:
   at Wox.Infrastructure.ShellLinkHelper.IShellLinkW.GetDescription(StringBuilder pszName, Int32 cchMaxName)
   at Wox.Infrastructure.ShellLinkHelper.RetrieveTargetPath(String path)
Exception source     : Wox.Infrastructure
Exception target site: Void GetDescription(System.Text.StringBuilder, Int32)
Exception HResult    : -2147467259
-------------------------- End exception --------------------------


[2024-10-03 08:12:59.2819] [INFO] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Storage\Win32ProgramRepository.cs::243]
Indexed 1731 win32 applications

[2024-10-03 08:12:59.2819] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Stopwatch.cs::44]
Microsoft.Plugin.Program.Main - Win32Program index cost <8542ms>

[2024-10-03 08:12:59.2819] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <Program> is <8850ms>

[2024-10-03 08:12:59.2819] [INFO] [D:\a\_work\1\s\src\modules\launcher\PowerLauncher\ViewModel\MainViewModel.cs::1333]
Refresh plugins overview

[2024-10-03 08:12:59.2819] [INFO] [D:\a\_work\1\s\src\modules\launcher\PowerLauncher\App.xaml.cs::157]
Begin PowerToys Run startup ----------------------------------------------------
Runtime info:
Version: 0.84.1.0
OS Version: Microsoft Windows NT 10.0.22631.0
IntPtr Length: 8
x64: True
End PowerToys Run startup ----------------------------------------------------  


[2024-10-03 08:12:59.2819] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Stopwatch.cs::44]
App.OnStartup - Startup cost <9974ms>

[2024-10-03 08:12:59.4592] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Stopwatch.cs::44]
EnvironmentHelper.GetProtectedEnvironmentVariables - Duration cost <0ms>

[2024-10-03 08:12:59.4592] [INFO] [D:\a\_work\1\s\src\modules\launcher\PowerLauncher\ViewModel\MainViewModel.cs::100]
RegisterHotkey()

[2024-10-03 08:12:59.4592] [INFO] [D:\a\_work\1\s\src\modules\launcher\PowerLauncher\ViewModel\MainViewModel.cs::853]
Set HotKey()

[2024-10-03 08:12:59.4592] [INFO] [D:\a\_work\1\s\src\modules\launcher\PowerLauncher\ViewModel\MainViewModel.cs::894]
Registered global hotkey

[2024-10-03 08:47:11.9765] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.People
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at WinRT.ExceptionHelpers.ThrowExceptionForHR(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Windows.ApplicationModel.Package.get_InstalledLocation()
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 08:47:13.2985] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Storage\PackageRepository.cs::88]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Windows.Storage.StorageFolder
		Exception thrown in called method: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
		Possible interpretation of the error: Could not find file 'C:\Program Files\WindowsApps\Microsoft.People_2021.2202.100.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
		Possible resolution: Not yet known

Exception full name  : System.IO.FileNotFoundException
Exception message    : Could not find file 'C:\Program Files\WindowsApps\Microsoft.People_2021.2202.100.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
Exception stack trace:
   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.Xml.XmlDownloadManager.GetStream(Uri uri, ICredentials credentials, IWebProxy proxy)
   at System.Xml.XmlTextReaderImpl.FinishInitUriString()
   at System.Xml.XmlReaderSettings.CreateReader(String inputUri, XmlParserContext inputContext)
   at System.Xml.Linq.XDocument.Load(String uri, LoadOptions options)
   at Microsoft.Plugin.Program.Programs.UWP.XmlNamespaces(String path)
   at Microsoft.Plugin.Program.Programs.UWP.InitializeAppInfo(String installedLocation)
   at Microsoft.Plugin.Program.Storage.PackageRepository.AddPackage(Package package)
Exception source     : System.Private.CoreLib
Exception target site: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
Exception HResult    : -2147024894
-------------------------- End exception --------------------------


[2024-10-03 08:48:33.7804] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception PythonSoftwareFoundation.Python.3.12
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Windows.ApplicationModel.Package.get_InstalledLocation()
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 08:48:54.3593] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.LanguageExperiencePackvi-VN
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Windows.ApplicationModel.Package.get_InstalledLocation()
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 08:50:28.1722] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception CanonicalGroupLimited.Ubuntu
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Windows.ApplicationModel.Package.get_InstalledLocation()
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 08:50:28.1722] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception CanonicalGroupLimited.Ubuntu
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Windows.ApplicationModel.Package.get_InstalledLocation()
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 08:50:28.1722] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception CanonicalGroupLimited.Ubuntu
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Windows.ApplicationModel.Package.get_InstalledLocation()
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 08:50:28.1796] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception CanonicalGroupLimited.Ubuntu
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Windows.ApplicationModel.Package.get_InstalledLocation()
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 08:50:28.1796] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception CanonicalGroupLimited.Ubuntu
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Windows.ApplicationModel.Package.get_InstalledLocation()
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 08:50:28.5939] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Storage\PackageRepository.cs::88]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Windows.Storage.StorageFolder
		Exception thrown in called method: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
		Possible interpretation of the error: Could not find file 'C:\Program Files\WindowsApps\CanonicalGroupLimited.Ubuntu_2404.1.66.0_neutral_~_79rhkp1fndgsc\AppxManifest.xml'.
		Possible resolution: Not yet known

Exception full name  : System.IO.FileNotFoundException
Exception message    : Could not find file 'C:\Program Files\WindowsApps\CanonicalGroupLimited.Ubuntu_2404.1.66.0_neutral_~_79rhkp1fndgsc\AppxManifest.xml'.
Exception stack trace:
   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.Xml.XmlDownloadManager.GetStream(Uri uri, ICredentials credentials, IWebProxy proxy)
   at System.Xml.XmlTextReaderImpl.FinishInitUriString()
   at System.Xml.XmlReaderSettings.CreateReader(String inputUri, XmlParserContext inputContext)
   at System.Xml.Linq.XDocument.Load(String uri, LoadOptions options)
   at Microsoft.Plugin.Program.Programs.UWP.XmlNamespaces(String path)
   at Microsoft.Plugin.Program.Programs.UWP.InitializeAppInfo(String installedLocation)
   at Microsoft.Plugin.Program.Storage.PackageRepository.AddPackage(Package package)
Exception source     : System.Private.CoreLib
Exception target site: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
Exception HResult    : -2147024894
-------------------------- End exception --------------------------


[2024-10-03 17:39:36.4375] [INFO] [D:\a\_work\1\s\src\modules\launcher\PowerLauncher\App.xaml.cs::206]
Application.Current.SessionEnding

[2024-10-03 17:39:36.4375] [INFO] [D:\a\_work\1\s\src\modules\launcher\PowerLauncher\App.xaml.cs::254]
Start PowerToys Run Exit----------------------------------------------------  

[2024-10-03 17:39:36.5136] [INFO] [D:\a\_work\1\s\src\modules\launcher\PowerLauncher\App.xaml.cs::97]
Runner with pid=24200 exited. Exiting PowerToys Run

[2024-10-03 17:39:36.5575] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Storage\JsonStorage`1.cs::139]
Saving cached data at <C:\Users\LENOVO LEGION\AppData\Local\Microsoft\PowerToys\PowerToys Run\Settings\QueryHistory.json>

[2024-10-03 17:39:36.5746] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Storage\JsonStorage`1.cs::139]
Saving cached data at <C:\Users\LENOVO LEGION\AppData\Local\Microsoft\PowerToys\PowerToys Run\Settings\UserSelectedRecord.json>

[2024-10-03 17:39:36.6027] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Storage\JsonStorage`1.cs::139]
Saving cached data at <C:\Users\LENOVO LEGION\AppData\Local\Microsoft\PowerToys\PowerToys Run\Settings\PowerToysRunSettings.json>

[2024-10-03 17:39:36.6027] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Storage\JsonStorage`1.cs::139]
Saving cached data at <C:\Users\LENOVO LEGION\AppData\Local\Microsoft\PowerToys\PowerToys Run\Settings\Plugins\Microsoft.Plugin.Folder\FolderSettings.json>

[2024-10-03 17:39:36.6188] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Storage\JsonStorage`1.cs::139]
Saving cached data at <C:\Users\LENOVO LEGION\AppData\Local\Microsoft\PowerToys\PowerToys Run\Settings\Plugins\Microsoft.Plugin.Indexer\IndexerSettings.json>

[2024-10-03 17:39:36.6188] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Storage\JsonStorage`1.cs::139]
Saving cached data at <C:\Users\LENOVO LEGION\AppData\Local\Microsoft\PowerToys\PowerToys Run\Settings\Plugins\Microsoft.Plugin.Program\ProgramPluginSettings.json>

[2024-10-03 17:39:36.6280] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Storage\JsonStorage`1.cs::139]
Saving cached data at <C:\Users\LENOVO LEGION\AppData\Local\Microsoft\PowerToys\PowerToys Run\Settings\Plugins\Microsoft.Plugin.Shell\ShellPluginSettings.json>

[2024-10-03 17:39:36.6280] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Storage\JsonStorage`1.cs::139]
Saving cached data at <C:\Users\LENOVO LEGION\AppData\Local\Microsoft\PowerToys\PowerToys Run\Settings\Plugins\Microsoft.Plugin.Uri\UriSettings.json>

[2024-10-03 17:39:36.6535] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Storage\JsonStorage`1.cs::139]
Saving cached data at <C:\Users\LENOVO LEGION\AppData\Local\Microsoft\PowerToys\PowerToys Run\Settings\ImageCache.json>

[2024-10-03 17:39:36.6676] [INFO] [D:\a\_work\1\s\src\modules\launcher\PowerLauncher\App.xaml.cs::265]
End PowerToys Run Exit ----------------------------------------------------  

[2024-10-03 17:39:36.6676] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Stopwatch.cs::44]
App.OnExit - Exit cost <225ms>

[2024-10-03 19:26:59.2341] [INFO] [D:\a\_work\1\s\src\modules\launcher\PowerLauncher\App.xaml.cs::55]
Starting PowerToys Run with PID=22668

[2024-10-03 19:26:59.2514] [INFO] [D:\a\_work\1\s\src\modules\launcher\PowerLauncher\App.xaml.cs::66]
Runner pid=5168

[2024-10-03 19:26:59.5570] [INFO] [D:\a\_work\1\s\src\modules\launcher\PowerLauncher\App.xaml.cs::108]
On Startup.

[2024-10-03 19:26:59.6261] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Stopwatch.cs::44]
ImageLoader.Initialize - Preload images cost <3ms>

[2024-10-03 19:26:59.6261] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Image\ImageLoader.cs::99]
Number of preload images is <109>, Images Number: 2, Unique Items 2

[2024-10-03 19:26:59.6943] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Stopwatch.cs::44]
|Wox.Infrastructure.Alphabet.Initialize|Preload pinyin cache <49ms>

[2024-10-03 19:26:59.6943] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Alphabet.cs::46]
Number of preload pinyin combination<0>

[2024-10-03 19:26:59.9776] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <Calculator> is <43ms>

[2024-10-03 19:26:59.9955] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <Folder> is <2ms>

[2024-10-03 19:27:00.0452] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <History> is <0ms>

[2024-10-03 19:27:00.0452] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <Windows Indexer> is <1ms>

[2024-10-03 19:27:00.0452] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <OneNote> is <0ms>

[2024-10-03 19:27:00.0452] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <PowerToys> is <0ms>

[2024-10-03 19:27:00.5505] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <Program> is <498ms>

[2024-10-03 19:27:00.5505] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <Registry> is <0ms>

[2024-10-03 19:27:00.5556] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <Service> is <1ms>

[2024-10-03 19:27:00.5556] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <Shell> is <5ms>

[2024-10-03 19:27:00.5662] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <System Commands> is <2ms>

[2024-10-03 19:27:00.5662] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <Time and date> is <0ms>

[2024-10-03 19:27:00.5662] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <Unit Converter> is <0ms>

[2024-10-03 19:27:00.5968] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <Windows Uri Handler> is <1ms>

[2024-10-03 19:27:00.5968] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <ValueGenerator> is <0ms>

[2024-10-03 19:27:00.6613] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <VS Code Workspaces> is <60ms>

[2024-10-03 19:27:00.6613] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <Web Search> is <0ms>

[2024-10-03 19:27:00.6666] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <Windows Settings> is <1ms>

[2024-10-03 19:27:00.6920] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <Windows Terminal> is <24ms>

[2024-10-03 19:27:00.6920] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::144]
Load cost for <Window Walker> is <0ms>

[2024-10-03 19:27:02.5371] [INFO] [D:\a\_work\1\s\src\modules\launcher\PowerLauncher\SettingsReader.cs::87]
Successfully read new settings. retryCount=1

[2024-10-03 19:27:02.6540] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Storage\JsonStorage`1.cs::139]
Saving cached data at <C:\Users\LENOVO LEGION\AppData\Local\Microsoft\PowerToys\PowerToys Run\Settings\Plugins\Microsoft.Plugin.Shell\ShellPluginSettings.json>

[2024-10-03 19:27:02.7922] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <Shell> is <5ms>

[2024-10-03 19:27:02.7922] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <Service> is <1ms>

[2024-10-03 19:27:02.7922] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <Registry> is <0ms>

[2024-10-03 19:27:02.7922] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <Calculator> is <43ms>

[2024-10-03 19:27:02.7922] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <Time and date> is <0ms>

[2024-10-03 19:27:02.7922] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <ValueGenerator> is <0ms>

[2024-10-03 19:27:02.7922] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <Unit Converter> is <0ms>

[2024-10-03 19:27:02.7922] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <Windows Uri Handler> is <1ms>

[2024-10-03 19:27:02.7922] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <History> is <0ms>

[2024-10-03 19:27:02.7922] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <System Commands> is <2ms>

[2024-10-03 19:27:02.7922] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\Common\DefaultBrowserInfo.cs::74]
I've tried updating the chosen Web Browser info at least once.

[2024-10-03 19:27:02.7922] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <Folder> is <4ms>

[2024-10-03 19:27:02.8007] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <PowerToys> is <8ms>

[2024-10-03 19:27:02.8007] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <Window Walker> is <0ms>

[2024-10-03 19:27:02.8007] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <Windows Indexer> is <21ms>

[2024-10-03 19:27:02.8229] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <Windows Settings> is <13ms>

[2024-10-03 19:27:02.8308] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <Web Search> is <39ms>

[2024-10-03 19:27:04.0206] [INFO] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Storage\PackageRepository.cs::111]
Indexed 45 packaged applications

[2024-10-03 19:27:04.0358] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Stopwatch.cs::44]
Microsoft.Plugin.Program.Main - Package index cost <1118ms>

[2024-10-03 19:27:04.5480] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\ShellLinkHelper.cs::169]
-------------------------- Begin exception --------------------------
Message: Failed to fetch description for C:\Windows\syswow64\odbcad32.exe, Error HRESULT E_FAIL has been returned from a call to a COM component.

Exception full name  : System.Runtime.InteropServices.COMException
Exception message    : Error HRESULT E_FAIL has been returned from a call to a COM component.
Exception stack trace:
   at Wox.Infrastructure.ShellLinkHelper.IShellLinkW.GetDescription(StringBuilder pszName, Int32 cchMaxName)
   at Wox.Infrastructure.ShellLinkHelper.RetrieveTargetPath(String path)
Exception source     : Wox.Infrastructure
Exception target site: Void GetDescription(System.Text.StringBuilder, Int32)
Exception HResult    : -2147467259
-------------------------- End exception --------------------------


[2024-10-03 19:27:04.5745] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\ShellLinkHelper.cs::169]
-------------------------- Begin exception --------------------------
Message: Failed to fetch description for C:\PROGRA~2\WINDOW~4\wmplayer.exe, Error HRESULT E_FAIL has been returned from a call to a COM component.

Exception full name  : System.Runtime.InteropServices.COMException
Exception message    : Error HRESULT E_FAIL has been returned from a call to a COM component.
Exception stack trace:
   at Wox.Infrastructure.ShellLinkHelper.IShellLinkW.GetDescription(StringBuilder pszName, Int32 cchMaxName)
   at Wox.Infrastructure.ShellLinkHelper.RetrieveTargetPath(String path)
Exception source     : Wox.Infrastructure
Exception target site: Void GetDescription(System.Text.StringBuilder, Int32)
Exception HResult    : -2147467259
-------------------------- End exception --------------------------


[2024-10-03 19:27:11.4253] [INFO] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Storage\Win32ProgramRepository.cs::243]
Indexed 1731 win32 applications

[2024-10-03 19:27:11.4268] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Stopwatch.cs::44]
Microsoft.Plugin.Program.Main - Win32Program index cost <8509ms>

[2024-10-03 19:27:11.4268] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Plugin\PluginPair.cs::56]
Total initialize cost for <Program> is <9133ms>

[2024-10-03 19:27:11.4268] [INFO] [D:\a\_work\1\s\src\modules\launcher\PowerLauncher\ViewModel\MainViewModel.cs::1333]
Refresh plugins overview

[2024-10-03 19:27:11.4268] [INFO] [D:\a\_work\1\s\src\modules\launcher\PowerLauncher\App.xaml.cs::157]
Begin PowerToys Run startup ----------------------------------------------------
Runtime info:
Version: 0.84.1.0
OS Version: Microsoft Windows NT 10.0.22631.0
IntPtr Length: 8
x64: True
End PowerToys Run startup ----------------------------------------------------  


[2024-10-03 19:27:11.4268] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Stopwatch.cs::44]
App.OnStartup - Startup cost <11857ms>

[2024-10-03 19:27:11.4268] [INFO] [D:\a\_work\1\s\src\modules\launcher\PowerLauncher\MainWindow.xaml.cs::88]
Send Run settings telemetry

[2024-10-03 19:27:11.5896] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Stopwatch.cs::44]
EnvironmentHelper.GetProtectedEnvironmentVariables - Duration cost <0ms>

[2024-10-03 19:27:11.5896] [INFO] [D:\a\_work\1\s\src\modules\launcher\PowerLauncher\ViewModel\MainViewModel.cs::100]
RegisterHotkey()

[2024-10-03 19:27:11.5896] [INFO] [D:\a\_work\1\s\src\modules\launcher\PowerLauncher\ViewModel\MainViewModel.cs::853]
Set HotKey()

[2024-10-03 19:27:11.5896] [INFO] [D:\a\_work\1\s\src\modules\launcher\PowerLauncher\ViewModel\MainViewModel.cs::894]
Registered global hotkey

[2024-10-03 20:07:05.3770] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Storage\ListRepository`1.cs::54]
Item Already Exists <ImageMagick (color depth=q16) (Ubuntu).lnk>

[2024-10-03 20:07:11.3853] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Storage\ListRepository`1.cs::54]
Item Already Exists <Install RELEASE (Ubuntu).lnk>

[2024-10-03 20:07:17.4010] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Storage\ListRepository`1.cs::54]
Item Already Exists <Tilix (Ubuntu).lnk>

[2024-10-03 20:07:23.4341] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Storage\ListRepository`1.cs::54]
Item Already Exists <xpdf (Ubuntu).lnk>

[2024-10-03 20:15:12.7415] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.YourPhone
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at WinRT.ExceptionHelpers.ThrowExceptionForHR(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Windows.ApplicationModel.Package.get_InstalledLocation()
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 20:15:13.7885] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Storage\PackageRepository.cs::88]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Windows.Storage.StorageFolder
		Exception thrown in called method: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
		Possible interpretation of the error: Could not find file 'C:\Program Files\WindowsApps\Microsoft.YourPhone_1.24082.137.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
		Possible resolution: Not yet known

Exception full name  : System.IO.FileNotFoundException
Exception message    : Could not find file 'C:\Program Files\WindowsApps\Microsoft.YourPhone_1.24082.137.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
Exception stack trace:
   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize)
   at System.Xml.XmlDownloadManager.GetStream(Uri uri, ICredentials credentials, IWebProxy proxy)
   at System.Xml.XmlTextReaderImpl.FinishInitUriString()
   at System.Xml.XmlReaderSettings.CreateReader(String inputUri, XmlParserContext inputContext)
   at System.Xml.Linq.XDocument.Load(String uri, LoadOptions options)
   at Microsoft.Plugin.Program.Programs.UWP.XmlNamespaces(String path)
   at Microsoft.Plugin.Program.Programs.UWP.InitializeAppInfo(String installedLocation)
   at Microsoft.Plugin.Program.Storage.PackageRepository.AddPackage(Package package)
Exception source     : System.Private.CoreLib
Exception target site: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
Exception HResult    : -2147024894
-------------------------- End exception --------------------------


[2024-10-03 21:53:49.6479] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception MicrosoftWindows.CrossDevice
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Windows.ApplicationModel.Package.get_InstalledLocation()
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:53:53.2075] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.DesktopAppInstaller
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Windows.ApplicationModel.Package.get_InstalledLocation()
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:53:54.4616] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Storage\PackageRepository.cs::88]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Windows.Storage.StorageFolder
		Exception thrown in called method: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
		Possible interpretation of the error: Could not find file 'C:\Program Files\WindowsApps\Microsoft.DesktopAppInstaller_2024.709.2344.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
		Possible resolution: Not yet known

Exception full name  : System.IO.FileNotFoundException
Exception message    : Could not find file 'C:\Program Files\WindowsApps\Microsoft.DesktopAppInstaller_2024.709.2344.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
Exception stack trace:
   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize)
   at System.Xml.XmlDownloadManager.GetStream(Uri uri, ICredentials credentials, IWebProxy proxy)
   at System.Xml.XmlTextReaderImpl.FinishInitUriString()
   at System.Xml.XmlReaderSettings.CreateReader(String inputUri, XmlParserContext inputContext)
   at System.Xml.Linq.XDocument.Load(String uri, LoadOptions options)
   at Microsoft.Plugin.Program.Programs.UWP.XmlNamespaces(String path)
   at Microsoft.Plugin.Program.Programs.UWP.InitializeAppInfo(String installedLocation)
   at Microsoft.Plugin.Program.Storage.PackageRepository.AddPackage(Package package)
Exception source     : System.Private.CoreLib
Exception target site: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
Exception HResult    : -2147024894
-------------------------- End exception --------------------------


[2024-10-03 21:54:04.1141] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.Getstarted
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Windows.ApplicationModel.Package.get_InstalledLocation()
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:54:04.1141] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.Getstarted
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Windows.ApplicationModel.Package.get_InstalledLocation()
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:54:04.1141] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.Getstarted
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Windows.ApplicationModel.Package.get_InstalledLocation()
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:54:04.1215] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.Getstarted
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Windows.ApplicationModel.Package.get_InstalledLocation()
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:54:04.1215] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.Getstarted
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Windows.ApplicationModel.Package.get_InstalledLocation()
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:54:04.6284] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Storage\PackageRepository.cs::88]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Windows.Storage.StorageFolder
		Exception thrown in called method: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
		Possible interpretation of the error: Could not find file 'C:\Program Files\WindowsApps\Microsoft.Getstarted_2021.2312.1.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
		Possible resolution: Not yet known

Exception full name  : System.IO.FileNotFoundException
Exception message    : Could not find file 'C:\Program Files\WindowsApps\Microsoft.Getstarted_2021.2312.1.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
Exception stack trace:
   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize)
   at System.Xml.XmlDownloadManager.GetStream(Uri uri, ICredentials credentials, IWebProxy proxy)
   at System.Xml.XmlTextReaderImpl.FinishInitUriString()
   at System.Xml.XmlReaderSettings.CreateReader(String inputUri, XmlParserContext inputContext)
   at System.Xml.Linq.XDocument.Load(String uri, LoadOptions options)
   at Microsoft.Plugin.Program.Programs.UWP.XmlNamespaces(String path)
   at Microsoft.Plugin.Program.Programs.UWP.InitializeAppInfo(String installedLocation)
   at Microsoft.Plugin.Program.Storage.PackageRepository.AddPackage(Package package)
Exception source     : System.Private.CoreLib
Exception target site: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
Exception HResult    : -2147024894
-------------------------- End exception --------------------------


[2024-10-03 21:54:12.2792] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception microsoft.microsoftskydrive
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Windows.ApplicationModel.Package.get_InstalledLocation()
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:54:12.7815] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Storage\PackageRepository.cs::88]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Windows.Storage.StorageFolder
		Exception thrown in called method: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
		Possible interpretation of the error: Could not find file 'C:\Program Files\WindowsApps\microsoft.microsoftskydrive_19.23.19.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
		Possible resolution: Not yet known

Exception full name  : System.IO.FileNotFoundException
Exception message    : Could not find file 'C:\Program Files\WindowsApps\microsoft.microsoftskydrive_19.23.19.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
Exception stack trace:
   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize)
   at System.Xml.XmlDownloadManager.GetStream(Uri uri, ICredentials credentials, IWebProxy proxy)
   at System.Xml.XmlTextReaderImpl.FinishInitUriString()
   at System.Xml.XmlReaderSettings.CreateReader(String inputUri, XmlParserContext inputContext)
   at System.Xml.Linq.XDocument.Load(String uri, LoadOptions options)
   at Microsoft.Plugin.Program.Programs.UWP.XmlNamespaces(String path)
   at Microsoft.Plugin.Program.Programs.UWP.InitializeAppInfo(String installedLocation)
   at Microsoft.Plugin.Program.Storage.PackageRepository.AddPackage(Package package)
Exception source     : System.Private.CoreLib
Exception target site: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
Exception HResult    : -2147024894
-------------------------- End exception --------------------------


[2024-10-03 21:54:14.9991] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.XboxGamingOverlay
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Windows.ApplicationModel.Package.get_InstalledLocation()
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:54:14.9991] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.XboxGamingOverlay
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Windows.ApplicationModel.Package.get_InstalledLocation()
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:54:15.0047] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.XboxGamingOverlay
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Windows.ApplicationModel.Package.get_InstalledLocation()
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:54:15.0047] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.XboxGamingOverlay
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Windows.ApplicationModel.Package.get_InstalledLocation()
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:54:15.0047] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.XboxGamingOverlay
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Windows.ApplicationModel.Package.get_InstalledLocation()
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:54:15.0047] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.XboxGamingOverlay
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Windows.ApplicationModel.Package.get_InstalledLocation()
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:54:15.7816] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Storage\PackageRepository.cs::88]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Windows.Storage.StorageFolder
		Exception thrown in called method: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
		Possible interpretation of the error: Could not find file 'C:\Program Files\WindowsApps\Microsoft.XboxGamingOverlay_7.224.9242.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
		Possible resolution: Not yet known

Exception full name  : System.IO.FileNotFoundException
Exception message    : Could not find file 'C:\Program Files\WindowsApps\Microsoft.XboxGamingOverlay_7.224.9242.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
Exception stack trace:
   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize)
   at System.Xml.XmlDownloadManager.GetStream(Uri uri, ICredentials credentials, IWebProxy proxy)
   at System.Xml.XmlTextReaderImpl.FinishInitUriString()
   at System.Xml.XmlReaderSettings.CreateReader(String inputUri, XmlParserContext inputContext)
   at System.Xml.Linq.XDocument.Load(String uri, LoadOptions options)
   at Microsoft.Plugin.Program.Programs.UWP.XmlNamespaces(String path)
   at Microsoft.Plugin.Program.Programs.UWP.InitializeAppInfo(String installedLocation)
   at Microsoft.Plugin.Program.Storage.PackageRepository.AddPackage(Package package)
Exception source     : System.Private.CoreLib
Exception target site: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
Exception HResult    : -2147024894
-------------------------- End exception --------------------------


[2024-10-03 21:54:16.2361] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception 28944Stefanvd.BatteryDeck
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Windows.ApplicationModel.Package.get_InstalledLocation()
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:54:22.0526] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.WindowsCalculator
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Windows.ApplicationModel.Package.get_InstalledLocation()
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:54:22.5736] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Storage\PackageRepository.cs::88]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Windows.Storage.StorageFolder
		Exception thrown in called method: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
		Possible interpretation of the error: Could not find file 'C:\Program Files\WindowsApps\Microsoft.WindowsCalculator_2021.2405.2.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
		Possible resolution: Not yet known

Exception full name  : System.IO.FileNotFoundException
Exception message    : Could not find file 'C:\Program Files\WindowsApps\Microsoft.WindowsCalculator_2021.2405.2.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
Exception stack trace:
   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize)
   at System.Xml.XmlDownloadManager.GetStream(Uri uri, ICredentials credentials, IWebProxy proxy)
   at System.Xml.XmlTextReaderImpl.FinishInitUriString()
   at System.Xml.XmlReaderSettings.CreateReader(String inputUri, XmlParserContext inputContext)
   at System.Xml.Linq.XDocument.Load(String uri, LoadOptions options)
   at Microsoft.Plugin.Program.Programs.UWP.XmlNamespaces(String path)
   at Microsoft.Plugin.Program.Programs.UWP.InitializeAppInfo(String installedLocation)
   at Microsoft.Plugin.Program.Storage.PackageRepository.AddPackage(Package package)
Exception source     : System.Private.CoreLib
Exception target site: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
Exception HResult    : -2147024894
-------------------------- End exception --------------------------


[2024-10-03 21:54:27.8182] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.Paint
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Windows.ApplicationModel.Package.get_InstalledLocation()
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:54:28.6875] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Storage\PackageRepository.cs::88]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Windows.Storage.StorageFolder
		Exception thrown in called method: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
		Possible interpretation of the error: Could not find file 'C:\Program Files\WindowsApps\Microsoft.Paint_11.2406.42.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
		Possible resolution: Not yet known

Exception full name  : System.IO.FileNotFoundException
Exception message    : Could not find file 'C:\Program Files\WindowsApps\Microsoft.Paint_11.2406.42.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
Exception stack trace:
   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.Xml.XmlDownloadManager.GetStream(Uri uri, ICredentials credentials, IWebProxy proxy)
   at System.Xml.XmlTextReaderImpl.FinishInitUriString()
   at System.Xml.XmlReaderSettings.CreateReader(String inputUri, XmlParserContext inputContext)
   at System.Xml.Linq.XDocument.Load(String uri, LoadOptions options)
   at Microsoft.Plugin.Program.Programs.UWP.XmlNamespaces(String path)
   at Microsoft.Plugin.Program.Programs.UWP.InitializeAppInfo(String installedLocation)
   at Microsoft.Plugin.Program.Storage.PackageRepository.AddPackage(Package package)
Exception source     : System.Private.CoreLib
Exception target site: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
Exception HResult    : -2147024894
-------------------------- End exception --------------------------


[2024-10-03 21:54:28.7907] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.XboxIdentityProvider
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Windows.ApplicationModel.Package.get_InstalledLocation()
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:54:29.2592] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Storage\PackageRepository.cs::88]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Windows.Storage.StorageFolder
		Exception thrown in called method: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
		Possible interpretation of the error: Could not find file 'C:\Program Files\WindowsApps\Microsoft.XboxIdentityProvider_12.115.1001.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
		Possible resolution: Not yet known

Exception full name  : System.IO.FileNotFoundException
Exception message    : Could not find file 'C:\Program Files\WindowsApps\Microsoft.XboxIdentityProvider_12.115.1001.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
Exception stack trace:
   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.Xml.XmlDownloadManager.GetStream(Uri uri, ICredentials credentials, IWebProxy proxy)
   at System.Xml.XmlTextReaderImpl.FinishInitUriString()
   at System.Xml.XmlReaderSettings.CreateReader(String inputUri, XmlParserContext inputContext)
   at System.Xml.Linq.XDocument.Load(String uri, LoadOptions options)
   at Microsoft.Plugin.Program.Programs.UWP.XmlNamespaces(String path)
   at Microsoft.Plugin.Program.Programs.UWP.InitializeAppInfo(String installedLocation)
   at Microsoft.Plugin.Program.Storage.PackageRepository.AddPackage(Package package)
Exception source     : System.Private.CoreLib
Exception target site: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
Exception HResult    : -2147024894
-------------------------- End exception --------------------------


[2024-10-03 21:54:29.3523] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.MicrosoftStickyNotes
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:54:29.9793] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Storage\PackageRepository.cs::88]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Windows.Storage.StorageFolder
		Exception thrown in called method: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
		Possible interpretation of the error: Could not find file 'C:\Program Files\WindowsApps\Microsoft.MicrosoftStickyNotes_6.1.2.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
		Possible resolution: Not yet known

Exception full name  : System.IO.FileNotFoundException
Exception message    : Could not find file 'C:\Program Files\WindowsApps\Microsoft.MicrosoftStickyNotes_6.1.2.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
Exception stack trace:
   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.Xml.XmlDownloadManager.GetStream(Uri uri, ICredentials credentials, IWebProxy proxy)
   at System.Xml.XmlTextReaderImpl.FinishInitUriString()
   at System.Xml.XmlReaderSettings.CreateReader(String inputUri, XmlParserContext inputContext)
   at System.Xml.Linq.XDocument.Load(String uri, LoadOptions options)
   at Microsoft.Plugin.Program.Programs.UWP.XmlNamespaces(String path)
   at Microsoft.Plugin.Program.Programs.UWP.InitializeAppInfo(String installedLocation)
   at Microsoft.Plugin.Program.Storage.PackageRepository.AddPackage(Package package)
Exception source     : System.Private.CoreLib
Exception target site: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
Exception HResult    : -2147024894
-------------------------- End exception --------------------------


[2024-10-03 21:54:34.3454] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.GamingApp
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:54:36.4314] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.MicrosoftSolitaireCollection
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:54:36.8367] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Storage\PackageRepository.cs::88]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Windows.Storage.StorageFolder
		Exception thrown in called method: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
		Possible interpretation of the error: Could not find file 'C:\Program Files\WindowsApps\Microsoft.MicrosoftSolitaireCollection_4.20.9120.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
		Possible resolution: Not yet known

Exception full name  : System.IO.FileNotFoundException
Exception message    : Could not find file 'C:\Program Files\WindowsApps\Microsoft.MicrosoftSolitaireCollection_4.20.9120.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
Exception stack trace:
   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.Xml.XmlDownloadManager.GetStream(Uri uri, ICredentials credentials, IWebProxy proxy)
   at System.Xml.XmlTextReaderImpl.FinishInitUriString()
   at System.Xml.XmlReaderSettings.CreateReader(String inputUri, XmlParserContext inputContext)
   at System.Xml.Linq.XDocument.Load(String uri, LoadOptions options)
   at Microsoft.Plugin.Program.Programs.UWP.XmlNamespaces(String path)
   at Microsoft.Plugin.Program.Programs.UWP.InitializeAppInfo(String installedLocation)
   at Microsoft.Plugin.Program.Storage.PackageRepository.AddPackage(Package package)
Exception source     : System.Private.CoreLib
Exception target site: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
Exception HResult    : -2147024894
-------------------------- End exception --------------------------


[2024-10-03 21:54:37.8141] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception 21676OptimiliaStudios.AquileReader
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:54:38.2068] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Storage\PackageRepository.cs::88]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Windows.Storage.StorageFolder
		Exception thrown in called method: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
		Possible interpretation of the error: Could not find file 'C:\Program Files\WindowsApps\21676OptimiliaStudios.AquileReader_1.1.45.0_neutral_~_k42naep6bwmrc\AppxManifest.xml'.
		Possible resolution: Not yet known

Exception full name  : System.IO.FileNotFoundException
Exception message    : Could not find file 'C:\Program Files\WindowsApps\21676OptimiliaStudios.AquileReader_1.1.45.0_neutral_~_k42naep6bwmrc\AppxManifest.xml'.
Exception stack trace:
   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.Xml.XmlDownloadManager.GetStream(Uri uri, ICredentials credentials, IWebProxy proxy)
   at System.Xml.XmlTextReaderImpl.FinishInitUriString()
   at System.Xml.XmlReaderSettings.CreateReader(String inputUri, XmlParserContext inputContext)
   at System.Xml.Linq.XDocument.Load(String uri, LoadOptions options)
   at Microsoft.Plugin.Program.Programs.UWP.XmlNamespaces(String path)
   at Microsoft.Plugin.Program.Programs.UWP.InitializeAppInfo(String installedLocation)
   at Microsoft.Plugin.Program.Storage.PackageRepository.AddPackage(Package package)
Exception source     : System.Private.CoreLib
Exception target site: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
Exception HResult    : -2147024894
-------------------------- End exception --------------------------


[2024-10-03 21:54:49.4086] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.WindowsSoundRecorder
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:54:49.8536] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Storage\PackageRepository.cs::88]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Windows.Storage.StorageFolder
		Exception thrown in called method: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
		Possible interpretation of the error: Could not find file 'C:\Program Files\WindowsApps\Microsoft.WindowsSoundRecorder_2021.2408.6.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
		Possible resolution: Not yet known

Exception full name  : System.IO.FileNotFoundException
Exception message    : Could not find file 'C:\Program Files\WindowsApps\Microsoft.WindowsSoundRecorder_2021.2408.6.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
Exception stack trace:
   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.Xml.XmlDownloadManager.GetStream(Uri uri, ICredentials credentials, IWebProxy proxy)
   at System.Xml.XmlTextReaderImpl.FinishInitUriString()
   at System.Xml.XmlReaderSettings.CreateReader(String inputUri, XmlParserContext inputContext)
   at System.Xml.Linq.XDocument.Load(String uri, LoadOptions options)
   at Microsoft.Plugin.Program.Programs.UWP.XmlNamespaces(String path)
   at Microsoft.Plugin.Program.Programs.UWP.InitializeAppInfo(String installedLocation)
   at Microsoft.Plugin.Program.Storage.PackageRepository.AddPackage(Package package)
Exception source     : System.Private.CoreLib
Exception target site: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
Exception HResult    : -2147024894
-------------------------- End exception --------------------------


[2024-10-03 21:54:50.6959] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.BingWeather
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:54:51.3331] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Storage\PackageRepository.cs::88]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Windows.Storage.StorageFolder
		Exception thrown in called method: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
		Possible interpretation of the error: Could not find file 'C:\Program Files\WindowsApps\Microsoft.BingWeather_4.53.62621.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
		Possible resolution: Not yet known

Exception full name  : System.IO.FileNotFoundException
Exception message    : Could not find file 'C:\Program Files\WindowsApps\Microsoft.BingWeather_4.53.62621.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
Exception stack trace:
   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.Xml.XmlDownloadManager.GetStream(Uri uri, ICredentials credentials, IWebProxy proxy)
   at System.Xml.XmlTextReaderImpl.FinishInitUriString()
   at System.Xml.XmlReaderSettings.CreateReader(String inputUri, XmlParserContext inputContext)
   at System.Xml.Linq.XDocument.Load(String uri, LoadOptions options)
   at Microsoft.Plugin.Program.Programs.UWP.XmlNamespaces(String path)
   at Microsoft.Plugin.Program.Programs.UWP.InitializeAppInfo(String installedLocation)
   at Microsoft.Plugin.Program.Storage.PackageRepository.AddPackage(Package package)
Exception source     : System.Private.CoreLib
Exception target site: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
Exception HResult    : -2147024894
-------------------------- End exception --------------------------


[2024-10-03 21:54:52.4175] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.ZuneMusic
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:54:52.9644] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Storage\PackageRepository.cs::88]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Windows.Storage.StorageFolder
		Exception thrown in called method: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
		Possible interpretation of the error: Could not find file 'C:\Program Files\WindowsApps\Microsoft.ZuneMusic_11.2408.4.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
		Possible resolution: Not yet known

Exception full name  : System.IO.FileNotFoundException
Exception message    : Could not find file 'C:\Program Files\WindowsApps\Microsoft.ZuneMusic_11.2408.4.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
Exception stack trace:
   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.Xml.XmlTextReaderImpl.FinishInitUriString()
   at System.Xml.XmlTextReaderImpl..ctor(String uriStr, XmlReaderSettings settings, XmlParserContext context, XmlResolver uriResolver)
   at System.Xml.Linq.XDocument.Load(String uri, LoadOptions options)
   at Microsoft.Plugin.Program.Programs.UWP.XmlNamespaces(String path)
   at Microsoft.Plugin.Program.Programs.UWP.InitializeAppInfo(String installedLocation)
   at Microsoft.Plugin.Program.Storage.PackageRepository.AddPackage(Package package)
Exception source     : System.Private.CoreLib
Exception target site: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
Exception HResult    : -2147024894
-------------------------- End exception --------------------------


[2024-10-03 21:55:03.3778] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.PowerAutomateDesktop
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:55:03.9010] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Storage\PackageRepository.cs::88]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Windows.Storage.StorageFolder
		Exception thrown in called method: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
		Possible interpretation of the error: Could not find file 'C:\Program Files\WindowsApps\Microsoft.PowerAutomateDesktop_11.2409.203.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
		Possible resolution: Not yet known

Exception full name  : System.IO.FileNotFoundException
Exception message    : Could not find file 'C:\Program Files\WindowsApps\Microsoft.PowerAutomateDesktop_11.2409.203.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
Exception stack trace:
   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.Xml.XmlTextReaderImpl.FinishInitUriString()
   at System.Xml.XmlTextReaderImpl..ctor(String uriStr, XmlReaderSettings settings, XmlParserContext context, XmlResolver uriResolver)
   at System.Xml.Linq.XDocument.Load(String uri, LoadOptions options)
   at Microsoft.Plugin.Program.Programs.UWP.XmlNamespaces(String path)
   at Microsoft.Plugin.Program.Programs.UWP.InitializeAppInfo(String installedLocation)
   at Microsoft.Plugin.Program.Storage.PackageRepository.AddPackage(Package package)
Exception source     : System.Private.CoreLib
Exception target site: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
Exception HResult    : -2147024894
-------------------------- End exception --------------------------


[2024-10-03 21:55:06.5145] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.ScreenSketch
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:55:07.0312] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Storage\PackageRepository.cs::88]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Windows.Storage.StorageFolder
		Exception thrown in called method: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
		Possible interpretation of the error: Could not find file 'C:\Program Files\WindowsApps\Microsoft.ScreenSketch_2022.2407.3.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
		Possible resolution: Not yet known

Exception full name  : System.IO.FileNotFoundException
Exception message    : Could not find file 'C:\Program Files\WindowsApps\Microsoft.ScreenSketch_2022.2407.3.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
Exception stack trace:
   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.Xml.XmlTextReaderImpl.FinishInitUriString()
   at System.Xml.XmlTextReaderImpl..ctor(String uriStr, XmlReaderSettings settings, XmlParserContext context, XmlResolver uriResolver)
   at System.Xml.Linq.XDocument.Load(String uri, LoadOptions options)
   at Microsoft.Plugin.Program.Programs.UWP.XmlNamespaces(String path)
   at Microsoft.Plugin.Program.Programs.UWP.InitializeAppInfo(String installedLocation)
   at Microsoft.Plugin.Program.Storage.PackageRepository.AddPackage(Package package)
Exception source     : System.Private.CoreLib
Exception target site: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
Exception HResult    : -2147024894
-------------------------- End exception --------------------------


[2024-10-03 21:55:07.5992] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.WindowsCamera
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:55:08.1293] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Storage\PackageRepository.cs::88]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Windows.Storage.StorageFolder
		Exception thrown in called method: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
		Possible interpretation of the error: Could not find file 'C:\Program Files\WindowsApps\Microsoft.WindowsCamera_2022.2408.1.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
		Possible resolution: Not yet known

Exception full name  : System.IO.FileNotFoundException
Exception message    : Could not find file 'C:\Program Files\WindowsApps\Microsoft.WindowsCamera_2022.2408.1.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
Exception stack trace:
   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.Xml.XmlTextReaderImpl.FinishInitUriString()
   at System.Xml.XmlTextReaderImpl..ctor(String uriStr, XmlReaderSettings settings, XmlParserContext context, XmlResolver uriResolver)
   at System.Xml.Linq.XDocument.Load(String uri, LoadOptions options)
   at Microsoft.Plugin.Program.Programs.UWP.XmlNamespaces(String path)
   at Microsoft.Plugin.Program.Programs.UWP.InitializeAppInfo(String installedLocation)
   at Microsoft.Plugin.Program.Storage.PackageRepository.AddPackage(Package package)
Exception source     : System.Private.CoreLib
Exception target site: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
Exception HResult    : -2147024894
-------------------------- End exception --------------------------


[2024-10-03 21:55:10.6923] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception E046963F.LenovoCompanion
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:55:11.1408] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Storage\PackageRepository.cs::88]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Windows.Storage.StorageFolder
		Exception thrown in called method: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
		Possible interpretation of the error: Could not find file 'C:\Program Files\WindowsApps\E046963F.LenovoCompanion_10.2406.36.0_neutral_~_k1h2ywk1493x8\AppxManifest.xml'.
		Possible resolution: Not yet known

Exception full name  : System.IO.FileNotFoundException
Exception message    : Could not find file 'C:\Program Files\WindowsApps\E046963F.LenovoCompanion_10.2406.36.0_neutral_~_k1h2ywk1493x8\AppxManifest.xml'.
Exception stack trace:
   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.Xml.XmlTextReaderImpl.FinishInitUriString()
   at System.Xml.XmlTextReaderImpl..ctor(String uriStr, XmlReaderSettings settings, XmlParserContext context, XmlResolver uriResolver)
   at System.Xml.Linq.XDocument.Load(String uri, LoadOptions options)
   at Microsoft.Plugin.Program.Programs.UWP.XmlNamespaces(String path)
   at Microsoft.Plugin.Program.Programs.UWP.InitializeAppInfo(String installedLocation)
   at Microsoft.Plugin.Program.Storage.PackageRepository.AddPackage(Package package)
Exception source     : System.Private.CoreLib
Exception target site: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
Exception HResult    : -2147024894
-------------------------- End exception --------------------------


[2024-10-03 21:55:14.8280] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.MicrosoftOfficeHub
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:55:15.5296] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Storage\PackageRepository.cs::88]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Windows.Storage.StorageFolder
		Exception thrown in called method: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
		Possible interpretation of the error: Could not find file 'C:\Program Files\WindowsApps\Microsoft.MicrosoftOfficeHub_18.2409.1102.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
		Possible resolution: Not yet known

Exception full name  : System.IO.FileNotFoundException
Exception message    : Could not find file 'C:\Program Files\WindowsApps\Microsoft.MicrosoftOfficeHub_18.2409.1102.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
Exception stack trace:
   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.Xml.XmlTextReaderImpl.FinishInitUriString()
   at System.Xml.XmlTextReaderImpl..ctor(String uriStr, XmlReaderSettings settings, XmlParserContext context, XmlResolver uriResolver)
   at System.Xml.Linq.XDocument.Load(String uri, LoadOptions options)
   at Microsoft.Plugin.Program.Programs.UWP.XmlNamespaces(String path)
   at Microsoft.Plugin.Program.Programs.UWP.InitializeAppInfo(String installedLocation)
   at Microsoft.Plugin.Program.Storage.PackageRepository.AddPackage(Package package)
Exception source     : System.Private.CoreLib
Exception target site: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
Exception HResult    : -2147024894
-------------------------- End exception --------------------------


[2024-10-03 21:55:16.9178] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.WindowsNotepad
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:55:17.4645] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Storage\PackageRepository.cs::88]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Windows.Storage.StorageFolder
		Exception thrown in called method: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
		Possible interpretation of the error: Could not find file 'C:\Program Files\WindowsApps\Microsoft.WindowsNotepad_11.2407.9.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
		Possible resolution: Not yet known

Exception full name  : System.IO.FileNotFoundException
Exception message    : Could not find file 'C:\Program Files\WindowsApps\Microsoft.WindowsNotepad_11.2407.9.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
Exception stack trace:
   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.Xml.XmlTextReaderImpl.FinishInitUriString()
   at System.Xml.XmlTextReaderImpl..ctor(String uriStr, XmlReaderSettings settings, XmlParserContext context, XmlResolver uriResolver)
   at System.Xml.Linq.XDocument.Load(String uri, LoadOptions options)
   at Microsoft.Plugin.Program.Programs.UWP.XmlNamespaces(String path)
   at Microsoft.Plugin.Program.Programs.UWP.InitializeAppInfo(String installedLocation)
   at Microsoft.Plugin.Program.Storage.PackageRepository.AddPackage(Package package)
Exception source     : System.Private.CoreLib
Exception target site: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
Exception HResult    : -2147024894
-------------------------- End exception --------------------------


[2024-10-03 21:55:55.9781] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.LanguageExperiencePackvi-VN
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:56:12.3104] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.WindowsAppRuntime.1.6
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:56:12.7148] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.WindowsAlarms
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:56:12.7148] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.WindowsAlarms
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:56:12.7402] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.WindowsAlarms
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:56:12.7425] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.WindowsAlarms
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:56:12.7425] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.WindowsAlarms
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:56:12.7425] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.WindowsAlarms
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:56:13.6149] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Storage\PackageRepository.cs::88]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Windows.Storage.StorageFolder
		Exception thrown in called method: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
		Possible interpretation of the error: Could not find file 'C:\Program Files\WindowsApps\Microsoft.WindowsAlarms_2021.2408.9.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
		Possible resolution: Not yet known

Exception full name  : System.IO.FileNotFoundException
Exception message    : Could not find file 'C:\Program Files\WindowsApps\Microsoft.WindowsAlarms_2021.2408.9.0_neutral_~_8wekyb3d8bbwe\AppxManifest.xml'.
Exception stack trace:
   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at System.Xml.XmlTextReaderImpl.FinishInitUriString()
   at System.Xml.XmlTextReaderImpl..ctor(String uriStr, XmlReaderSettings settings, XmlParserContext context, XmlResolver uriResolver)
   at System.Xml.Linq.XDocument.Load(String uri, LoadOptions options)
   at Microsoft.Plugin.Program.Programs.UWP.XmlNamespaces(String path)
   at Microsoft.Plugin.Program.Programs.UWP.InitializeAppInfo(String installedLocation)
   at Microsoft.Plugin.Program.Storage.PackageRepository.AddPackage(Package package)
Exception source     : System.Private.CoreLib
Exception target site: Microsoft.Win32.SafeHandles.SafeFileHandle CreateFile(System.String, System.IO.FileMode, System.IO.FileAccess, System.IO.FileShare, System.IO.FileOptions)
Exception HResult    : -2147024894
-------------------------- End exception --------------------------


[2024-10-03 21:56:13.8043] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception Microsoft.WindowsAppRuntime.1.6
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 21:56:14.1181] [ERROR] [D:\a\_work\1\s\src\modules\launcher\Plugins\Microsoft.Plugin.Program\Programs\PackageWrapper.cs::56]
-------------------------- Begin exception --------------------------
Message: 	Error status: UNKNOWN
		Program path: Path could not be determined
		Exception thrown in called method: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
		Possible interpretation of the error: Exception MicrosoftCorporationII.WinAppRuntime.Singleton
		Possible resolution: Not yet known

Exception full name  : System.ArgumentException
Exception message    : Value does not fall within the expected range.
Exception stack trace:
   at WinRT.ExceptionHelpers.<ThrowExceptionForHR>g__Throw|39_0(Int32 hr)
   at ABI.Windows.ApplicationModel.IPackageMethods.get_InstalledLocation(IObjectReference _obj)
   at Microsoft.Plugin.Program.Programs.PackageWrapper.GetWrapperFromPackage(Package package)
Exception source     : WinRT.Runtime
Exception target site: Void <ThrowExceptionForHR>g__Throw|39_0(Int32)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------


[2024-10-03 22:48:39.8668] [INFO] [D:\a\_work\1\s\src\modules\launcher\PowerLauncher\App.xaml.cs::206]
Application.Current.SessionEnding

[2024-10-03 22:48:39.8668] [INFO] [D:\a\_work\1\s\src\modules\launcher\PowerLauncher\App.xaml.cs::254]
Start PowerToys Run Exit----------------------------------------------------  

[2024-10-03 22:48:39.9091] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Storage\JsonStorage`1.cs::139]
Saving cached data at <C:\Users\LENOVO LEGION\AppData\Local\Microsoft\PowerToys\PowerToys Run\Settings\QueryHistory.json>

[2024-10-03 22:48:39.9217] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Storage\JsonStorage`1.cs::139]
Saving cached data at <C:\Users\LENOVO LEGION\AppData\Local\Microsoft\PowerToys\PowerToys Run\Settings\UserSelectedRecord.json>

[2024-10-03 22:48:39.9513] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Storage\JsonStorage`1.cs::139]
Saving cached data at <C:\Users\LENOVO LEGION\AppData\Local\Microsoft\PowerToys\PowerToys Run\Settings\PowerToysRunSettings.json>

[2024-10-03 22:48:39.9513] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Storage\JsonStorage`1.cs::139]
Saving cached data at <C:\Users\LENOVO LEGION\AppData\Local\Microsoft\PowerToys\PowerToys Run\Settings\Plugins\Microsoft.Plugin.Folder\FolderSettings.json>

[2024-10-03 22:48:39.9616] [INFO] [D:\a\_work\1\s\src\modules\launcher\PowerLauncher\App.xaml.cs::97]
Runner with pid=5168 exited. Exiting PowerToys Run

[2024-10-03 22:48:39.9616] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Storage\JsonStorage`1.cs::139]
Saving cached data at <C:\Users\LENOVO LEGION\AppData\Local\Microsoft\PowerToys\PowerToys Run\Settings\Plugins\Microsoft.Plugin.Indexer\IndexerSettings.json>

[2024-10-03 22:48:39.9859] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Storage\JsonStorage`1.cs::139]
Saving cached data at <C:\Users\LENOVO LEGION\AppData\Local\Microsoft\PowerToys\PowerToys Run\Settings\Plugins\Microsoft.Plugin.Program\ProgramPluginSettings.json>

[2024-10-03 22:48:39.9905] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Storage\JsonStorage`1.cs::139]
Saving cached data at <C:\Users\LENOVO LEGION\AppData\Local\Microsoft\PowerToys\PowerToys Run\Settings\Plugins\Microsoft.Plugin.Shell\ShellPluginSettings.json>

[2024-10-03 22:48:39.9905] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Storage\JsonStorage`1.cs::139]
Saving cached data at <C:\Users\LENOVO LEGION\AppData\Local\Microsoft\PowerToys\PowerToys Run\Settings\Plugins\Microsoft.Plugin.Uri\UriSettings.json>

[2024-10-03 22:48:40.0185] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Storage\JsonStorage`1.cs::139]
Saving cached data at <C:\Users\LENOVO LEGION\AppData\Local\Microsoft\PowerToys\PowerToys Run\Settings\ImageCache.json>

[2024-10-03 22:48:40.0354] [INFO] [D:\a\_work\1\s\src\modules\launcher\PowerLauncher\App.xaml.cs::265]
End PowerToys Run Exit ----------------------------------------------------  

[2024-10-03 22:48:40.0354] [INFO] [D:\a\_work\1\s\src\modules\launcher\Wox.Infrastructure\Stopwatch.cs::44]
App.OnExit - Exit cost <167ms>

[2024-10-03 22:48:40.0839] [INFO] [D:\a\_work\1\s\src\modules\launcher\PowerLauncher\App.xaml.cs::200]
Application.Current.Exit

